javascript/i18n/phonenumbers/phonemetadata.pb.js

```diff
@@ -490,6 +504,130 @@ i18n.phonenumbers.PhoneNumberDesc.prototype.clearPossibleNumberPattern = functio
 
 
 /**
+ * Gets the value of the possible_length field at the index given.
+ * @param {number} index The index to lookup.
+ * @return {?number} The value.
+ */
+i18n.phonenumbers.PhoneNumberDesc.prototype.getPossibleLength = function(index) {
+  return /** @type {?number} */ (this.get$Value(9, index));
+};
+
+
+/**
+ * Gets the value of the possible_length field at the index given or the default value if not set.
+ * @param {number} index The index to lookup.
+ * @return {number} The value.
+ */
+i18n.phonenumbers.PhoneNumberDesc.prototype.getPossibleLengthOrDefault = function(index) {
```

Comment:
To reduce diffs with auto-generated js file. This file is the same as what protoc would generate now, just checked in.